source/utils/Helper.h

```diff
@@ -30,6 +30,10 @@
 #include <vector>
 #include <cstdint>
 
+#ifdef _MSC_VER
+    #define snprintf _snprintf
```

Comment:
I have to disagree here. I'm using snprintf to write a timestamp, while the old code created a static time locale, a stringstream, imbued the stream with the locale, used that to format a timestamp and then passed the resulting string along. This introduces several extra dynamic memory allocations in a method that should be as low overhead as possible.